### ISSUE TITLE
s/sever/server/

### ISSUE THREAD

**Issue by twosider** [fasterthanlime/feedback#168]
**Where's the problem at?**
I'm link challenged. It was today's article on eMacs and rustup.

**No, seriously, where is it?**

It seemed odd that the Emacs ecosystem would lack such functionality, so I looked it up: rustic says about automatic **sever** installation that lsp-mode provides this feature, but eglot doesn't, and then it says "Install rust-analyzer manually".

Copy/paste the paragraph the problem is in, maybe highlight what's wrong?

**Describe the issue and a fix if you know it**
In plain English, explain why this is wrong, and how to fix it (if you know how).
